src/sentry/integrations/utils/code_mapping.py

```diff
@@ -81,8 +106,7 @@ def __init__(self, frame_file_path: str) -> None:
             not frame_file_path
             or frame_file_path[0] in ["[", "<", "/"]
             or frame_file_path.find(" ") > -1
-            or frame_file_path.find("\\") > -1  # Windows support
-            or frame_file_path.find("/") == -1
+            or (frame_file_path.find("/") == -1 and not frame_file_path.find("\\") > -1)
```

Comment:
Since we now want to include Windows support, we can allow for paths with `\\` and no `/`

Reply:
The PR for the Windows tag is [here](https://github.com/getsentry/sentry/pull/42846)